ext/nnstreamer/tensor_filter/tensor_filter_onnxruntime.cc

```diff
@@ -0,0 +1,564 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * NNStreamer tensor_filter, sub-plugin for onnxruntime
+ * Copyright (C) 2023 Suyeon Kim <suyeon5.kim@samsung.com>
+ */
+/**
+ * @file	tensor_filter_onnxruntime.cc
+ * @date	30 Oct 2023
+ * @brief	NNStreamer tensor-filter sub-plugin for ONNXRuntime
+ * @see		http://github.com/nnstreamer/nnstreamer
+ * @see   https://onnxruntime.ai/
+ * @author	Suyeon Kim <suyeon5.kim@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ * This is the per-NN-framework plugin (onnxruntime) for tensor_filter.
+ *
+ * @todo This supports float32 input output only. Do support others.
+ * @todo This supports only CPU runtime on linux-x86_64. Do support others.
```

Comment:
```
@todo Only float32 is allowed for input/output. Other types are NYI.
@todo Only CPU is supported. GPU and other hardware support is NYI.
```

Anyway, in the future, you can add hardware support with the info given with ```accelerator``` property for tensor_filter.

Reply:
Thanks. updated it.